client/daemon.go

```diff
@@ -101,7 +102,7 @@ func (d *daemon) loop(refresh time.Duration) {
 		} else {
 			d.successCount++
 		}
-		logf("Update of keys completed after %d ms", time.Since(start).Milliseconds())
+		logf("Update of keys completed after %d ms", time.Since(start)*time.Millisecond)
```

Comment:
I don't believe this is needed.

Reply:
When I said this I meant the `*`, the .Milli() function should be sufficient without any change.